build.sbt

```diff
@@ -185,7 +185,7 @@ lazy val common = (project in file("common"))
     )
 
 lazy val libApi = (project in file("lib-api"))
-    .dependsOn(common % allConfigDependency)
+    .dependsOn(javaClientGenerated % allConfigDependency, common % allConfigDependency)
```

Comment:
Interfaces should be abstract and can only depend on common module. If the changes require other dependencies, then there is something wrong with the changes.